test/mailers/previews/user_notifier_mailer_preview.rb

```diff
@@ -0,0 +1,8 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_notifier_mailer
+class UserNotifierMailerPreview < ActionMailer::Preview
```

Comment:
I had made the changes. but i think i uploaded before saving that. It appeared when i used "git status". Sorry, rookie mistake